components/back-link/src/index.js

```diff
@@ -1,56 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
-import { BLACK, YELLOW } from 'govuk-colours';
+import { BLACK } from 'govuk-colours';
 import { withWhiteSpace } from '@govuk-react/hoc';
-import {
-  FONT_SIZE,
-  LINE_HEIGHT,
-  MEDIA_QUERIES,
-  NTA_LIGHT,
-} from '@govuk-react/constants';
+import { SPACING_POINTS } from '@govuk-react/constants';
+import { link, typography } from '@govuk-react/lib';
 
-const Anchor = styled('a')({
-  fontFamily: NTA_LIGHT,
-  WebkitFontSmoothing: 'antialiased',
-  MozOsxFontSmoothing: 'grayscale',
-  fontWeight: 400,
-  fontSize: FONT_SIZE.SIZE_14,
-  lineHeight: LINE_HEIGHT.SIZE_14,
-  [MEDIA_QUERIES.LARGESCREEN]: {
-    fontSize: FONT_SIZE.SIZE_16,
-    lineHeight: LINE_HEIGHT.SIZE_16,
-  },
-  display: 'inline-block',
-  position: 'relative',
-  marginTop: '15px',
-  paddingLeft: '14px',
-  border: 0,
-  backgroundColor: 'transparent',
-  color: `${BLACK}`,
-  borderBottom: `1px solid ${BLACK}`,
-  textDecoration: 'none',
-  '::before': {
-    display: 'block',
-    width: 0,
-    height: 0,
-    borderStyle: 'solid',
-    borderColor: 'transparent',
-    clipPath: 'polygon(0% 50%, 100% 100%, 100% 0%)',
-    borderWidth: '5px 6px 5px 0',
-    borderRightColor: `${BLACK}`,
-    content: "''",
-    position: 'absolute',
-    top: '-1px',
-    bottom: '1px',
-    left: 0,
-    margin: 'auto',
-  },
-  ':focus': {
-    backgroundColor: `${YELLOW}`,
-    outline: `3px solid ${YELLOW}`,
+const Anchor = styled('a')(
+  typography.font({ size: 16 }),
+  link.common(),
+  link.styleText,
+  {
+    display: 'inline-block',
+    position: 'relative',
+    // margins here are not responsive, hence why they're not specified using withWhiteSpace
+    marginTop: SPACING_POINTS[3],
+    marginBottom: SPACING_POINTS[3],
+    paddingLeft: '14px',
+    borderBottom: `1px solid ${BLACK}`,
+    textDecoration: 'none',
+    '::before': {
+      // TODO use an equivalent of govuk-shape-arrow here
```

Comment:
can we merge with this TODO in place?

Reply:
yeah i think so

govuk-frontend has a sass mixin for creating arrow shapes via borders called `govuk-shape-arrow` - essentially the code we have here is an in-lined rendered-out version of a call to `govuk-shape-arrow` (and essentially as it had been before).  i didn't want to go through the process of working out a lib and translation of this sass mixin at this time - feeling instead that it would be better to leave that to a later PR.

i'll add an issue to the backlog